juju/juju
# Add support for relation egress networks in state model

## Description of change

This is smaller than it looks - the PR doesn't reflect that state/relationingress.go is renamed to state/relationnetworks.go.

The relation ingress model in state is extended to support egress networks as well. Tweaks to the ingress network watcher were needed to filter out egress networks.

## QA steps

Run up a CMR scenario and ensure the expected ports are opened in the offering model.
